logo.svg

```diff
@@ -1,21 +1 @@
-  <path transform="matrix(.26458 0 0 .26458 0 197)" d="m274.7 142.42c2.511 3.0496 4.0254 6.9574 4.0254 11.236v88.473a17.695 17.695 0 0 1 -17.695 17.695h-141.56a17.695 17.695 0 0 1 -11.936 -4.6562l-0.01172 0.01171 118.84 118.84a188.98 188.98 0 0 0 144.69 -135.24l-96.354-96.355z" fill="url(#linearGradient4883)"/>
-  <g fill="#42a5f5">
-   <circle cx="50" cy="247" r="0"/>
-   <circle cx="50" cy="247" r="0"/>
-   <circle cx="50" cy="247" r="0"/>
-   <circle cx="50" cy="247" r="0"/>
-   <circle cx="50" cy="247" r="0"/>
-  </g>
- </g>
-</svg>
+<svg width="32" height="32" version="1.1" viewBox="0 0 8.4667 8.4667" xml:space="preserve" xmlns="http://www.w3.org/2000/svg" xmlns:cc="http://creativecommons.org/ns#" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:xlink="http://www.w3.org/1999/xlink"><title>Material Icon Theme</title><defs><linearGradient id="linearGradient7" x1="42.788" x2="76.355" y1="229.53" y2="278.15" gradientUnits="userSpaceOnUse"><stop stop-opacity=".20392" offset="0"/><stop stop-color="#36343d" stop-opacity=".019608" offset="1"/></linearGradient></defs><g transform="matrix(.084514 0 0 .084514 0 -16.649)"><circle transform="rotate(.017022)" cx="50.164" cy="247.08" r="50.09" fill="#01579b" stroke-width="1.0018"/><path d="m73.692 234.14-0.006114 0.00612a3.5099 3.5099 0 0 1 0.97832 2.4214l-0.006114 24.568a3.5099 3.5099 0 0 1-3.5159 3.5097l-42.117-0.01223a3.5099 3.5099 0 0 1-2.3174-0.92941l-0.030572 0.03057 8.7132 8.7071 23.871 23.877a50.09 50.09 0 0 0 26.237-13.8 50.09 50.09 0 0 0 13.207-23.364l-25.015-25.015z" fill="url(#linearGradient7)" fill-opacity=".28663" stroke-width="0"/><path d="m46.311 232.24-2.2598-1.882a3.5099 3.5099 0 0 0-2.2461-0.81497l-12.769-4e-3a3.5099 3.5099 0 0 0-3.5109 3.5088l-0.0083 28.079a3.5099 3.5099 0 0 0 3.5092 3.5106l42.118 0.0128a3.5099 3.5099 0 0 0 3.5109-3.5088l0.0073-24.569a3.5099 3.5099 0 0 0-3.5092-3.5106l-22.597-7e-3a3.5099 3.5099 0 0 1-2.2461-0.81496" fill="#fafafa" stroke-width="1.7549"/></g><metadata><rdf:RDF><cc:Work rdf:about=""><dc:title>Material Icon Theme</dc:title><dc:creator><cc:Agent><dc:title>Philipp Kief</dc:title></cc:Agent></dc:creator><dc:publisher><cc:Agent><dc:title>Material Extensions</dc:title></cc:Agent></dc:publisher><cc:license rdf:resource="http://creativecommons.org/licenses/by/4.0/"/></cc:Work><cc:License rdf:about="http://creativecommons.org/licenses/by/4.0/"><cc:permits rdf:resource="http://creativecommons.org/ns#Reproduction"/><cc:permits rdf:resource="http://creativecommons.org/ns#Distribution"/><cc:requires rdf:resource="http://creativecommons.org/ns#Notice"/><cc:requires rdf:resource="http://creativecommons.org/ns#Attribution"/><cc:permits rdf:resource="http://creativecommons.org/ns#DerivativeWorks"/></cc:License></rdf:RDF></metadata></svg>
```

Comment:
Do we really need metadata?

Reply:
I think people should notice that the logo has a license and that they should give appropriate credit if they copy this extension to somewhere else.